src/core/quick-actions.ts

```diff
@@ -52,7 +52,7 @@ const quickActions = {
   'avdump-mismatched-files': {
     name: 'AVDump Mismatched Files',
     function: 'AVDumpMismatchedFiles',
-    info: 'Used by the \'Unrecognised\' utility to find files in your library that don\'t match to a file in AniDB.',
+    info: 'Used by the \'Unrecognized\' utility to find files in your library that don\'t match to a file in AniDB.',
```

Comment:
Tbh i prefer non American spellings

Reply:
But the utility itself is also written with Z 😅 I never know there was a BE variant with "s" at all

Reply:
Indian English says "unrecognized" is the correct one.
I have spoken 🙃

Reply:
🤷‍♂️

Reply:
IMO, the way it's said (American) is the correct spelling. I'm sure there's an accent that refutes that, but meh. Every accent I've heard still uses a Zed

Reply:
I'll just borrow @krbrs's message from discord and post it here;

> Unrecognized and unrecognised are both English terms. Unrecognized is predominantly used in 🇺🇸 American (US) English ( en-US ) while unrecognised is predominantly used in 🇬🇧 British English (used in UK/AU/NZ) ( en-GB ). In the United States, there is a preference for "unrecognized" over "unrecognised" (98 to 2).

Also, I'll let @hidden4003 decide what to do.

Reply:
Looks like unrecognized is the majority.